paddle/fluid/operators/mkldnn/conv_mkldnn_op.cc

```diff
@@ -449,22 +460,22 @@ class ConvMKLDNNOpKernel : public paddle::framework::OpKernel<T> {
         bias_tz = paddle::framework::vectorize2int(bias->dims());
         auto bias_md = platform::MKLDNNMemDesc(bias_tz, memory::data_type::s32,
                                                memory::format::x);
-        conv_pd = ConvFwdPrimitiveDesc(src_md, weights_md, bias_md, dst_md,
-                                       strides, paddings, mkldnn_engine,
-                                       fuse_relu, fuse_residual_conn,
-                                       output_shift_scale, sum_scale, is_test);
+
+        conv_pd = ConvFwdPrimitiveDesc(
+            src_md, weights_md, bias_md, dst_md, strides, paddings,
+            mkldnn_engine, fuse_relu, fuse_residual_conn, false, 0.0,
```

Comment:
- false means `fuse_brelu`?
- 0.0 means `fuse_brelu_threshold`?
It's hard to understand.

Reply:
They're placeholder as int8 fusion was not enabling if you read the previous conversation.  It will be removed once the int8 is enabled.

Reply:
You can write
```
false /*xxx */, 0.0 /*xxx*/
```

Reply:
> You can write
> 
> ```
> false /*xxx */, 0.0 /*xxx*/
> ```

updated the code!

Reply:
Got it. But the default of `fuse_brelu_threshold` is 6.0, why here is 0.0?

Reply:
> It will be removed once the int8 is enabled.

@guomingz @lidanqing-intel I do not see it be removed in https://github.com/PaddlePaddle/Paddle/pull/17546/commits/c9becf4bef8c79134a8fddc8b8f89d81d3aba160

Reply:
> Got it. But the default of `fuse_brelu_threshold` is 6.0, why here is 0.0?

0.0 implys the brelu fusion is not enabled.